<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Blackjack</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #2E8B57;
}
QLabel {
    color: #FFFFFF;
    font-weight: bold;
    font-size: 16px;
}
QPushButton {
    background-color: #008000;
    color: #FFFFFF;
    border: 2px solid #006400;
    border-radius: 8px;
    padding: 8px 16px;
    font-size: 14px;
}
QPushButton:hover {
    background-color: #00A000;
}
QPushButton:pressed {
    background-color: #006400;
    border-style: inset;
}
QStatusBar {
    background-color: #333333;
    color: #DDDDDD;
    font-size: 12px;
}
#gameStatusLabel {
    color: #FFD700;
    font-size: 18px;
    font-style: italic;
}
#gameBoardFrame {
    border: 5px solid #8B4513;
    border-radius: 15px;
    background-color: #2E8B57;
}
#cardPlaceholder {
    background-color: rgba(255, 255, 255, 50);
    border: 1px solid #FFFFFF;
    border-radius: 5px;
    min-width: 80px;
    min-height: 120px;
    max-width: 80px;
    max-height: 120px;
}
#dealerDisplayFrame, #playerDisplayFrame {
    border: 3px solid #FFFFFF;
    background-color: rgba(0, 0, 0, 50);
    border-radius: 10px;
    padding: 5px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>new game </string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="topSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QFrame" name="gameBoardFrame">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="gameBoardLayout">
       <item>
        <widget class="QLabel" name="dealerLabel">
         <property name="styleSheet">
          <string notr="true">font-size: 18px;</string>
         </property>
         <property name="text">
          <string>Dealer's Hand</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="dealerDisplayFrame">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="dealerCardLayout">
          <property name="spacing">
           <number>10</number>
          </property>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="middleSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="playerLabel">
         <property name="styleSheet">
          <string notr="true">font-size: 18px;</string>
         </property>
         <property name="text">
          <string>Player's Hand</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="playerDisplayFrame">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="playerCardLayout">
          <property name="spacing">
           <number>10</number>
          </property>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="bottomSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="controlsLayout">
      <item>
       <widget class="QPushButton" name="hitButton">
        <property name="text">
         <string>Hit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="standButton">
        <property name="text">
         <string>Stand</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="doubleButton">
        <property name="text">
         <string>Double</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="splitButton">
        <property name="text">
         <string>Split</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="statusLayout">
      <item>
       <widget class="QLabel" name="balanceLabel">
        <property name="text">
         <string>Balance: $10000</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="gameStatusLabel">
        <property name="text">
         <string>Place your bet!</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="betLabel">
        <property name="text">
         <string>Current Bet: $0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
